pom.xml

```diff
@@ -121,7 +120,6 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <org.apache.ws.commons.version>1.0.1</org.apache.ws.commons.version>
     <org.artofsolving.jodconverter.version>3.0-eXo03</org.artofsolving.jodconverter.version>
     <org.aspectj.version>1.8.8</org.aspectj.version>
-    <org.bouncycastle.version>1.45</org.bouncycastle.version>
```

Comment:
https://github.com/search?l=Maven+POM&q=org%3AMeeds-io+%22org.bouncycastle%22&type=Code
we use bouncycastle, isn't it ?

Reply:
Hello. We had dependencies on it, but not used.
Nevertheless, tika have a transitive dependency on it, and I keep this one, so bcprov and bcmail jar are present in the final plf zip (in version 1.67)